.github/workflows/workflow.yml

```diff
@@ -248,7 +271,50 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0 # Fetch all the tags
+      - name: Get version
+        run: |
+          chcp 65001 #set code page to utf-8
+          echo ("PACKAGE_VERSION=" + (python3 ./setup.py --version) ) >> $env:GITHUB_ENV
+          echo "Version setup.py: ${{ env.PACKAGE_VERSION }}"
+      - name: Download conda build
+        uses: actions/download-artifact@v3
+        with:
+          name: conda-build-${{ env.PACKAGE_VERSION }}
+          path: conda-build-tmp
       - name: Install with conda
-        run: conda install -c openfisca openfisca-france-dev
+        run: |
+          conda install -c ./conda-build-tmp/noarch/openfisca-france-dev-${{ env.PACKAGE_VERSION }}-py_0.tar.bz2  openfisca-france-dev
       - name: openfisca test
         run: openfisca test --country-package openfisca_france tests
+
+  publish-to-conda:
+    runs-on: "ubuntu-latest"
+    needs: [ deploy, test-on-windows ]
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: "3.7.9"
+          # Add conda-forge for OpenFisca-Core
+          channels: conda-forge
+          activate-environment: true
+      - name: Get source code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch all the tags
```

Comment:
This significantly slows down the cloning process. Please explain in the comment not _what_ that does but _why_ it is done 🙂

Reply:
It's done like that everywhere in the OF workflow (6 times) so I just continue to do it.
I remove it in all place ?